src/backport.ts

```diff
@@ -274,25 +284,27 @@ const backport = async ({
       number,
     });
     const head = getHead({ base, number });
-    const labels = getLabels({
-      base,
-      labels: originalLabels
-        .map(({ name }) => name)
-        .filter((label) => !labelRegExp.test(label)),
-    });
+    const labels = originalLabels
+      .map((label) => label.name)
+      .filter((label) => !labelRegExp.test(label));
+    labels.push(`backported-to-${base}`);
```

Comment:
I think we also want a generic `backport` so that we can easily search for all backports, WDYT?

Reply:
I would say `backports` or `backported` because backport is a key word and will likely result in the gh action to get triggered and rerun